curriculum/challenges/english/02-javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/sum-all-numbers-in-a-range.english.md

```diff
@@ -11,7 +11,7 @@ We'll pass you an array of two numbers. Return the sum of those two numbers plus
 
 Remember to use <a href='http://forum.freecodecamp.org/t/how-to-get-help-when-you-are-stuck/19514' target='_blank'>Read-Search-Ask</a> if you get stuck. Try to pair program. Write your own code.
  
-### Example
+> Example:
```

Comment:
You don't need the `>` before "Example". You can just begin the sentence directly, like "For example, ...".

Also, since the code in your example works inline too, you can put everything on one line, like so: 

> For example, <code>sumAll([4,1])</code> should return <code>10</code> because sum of all the numbers between 1 and 4 (both inclusive) is <code>10</code>.

This follows the style used across most of the other challenges, like [this one](https://learn.freecodecamp.org/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/sum-all-odd-fibonacci-numbers/). 

---

Note - Do remove the `>` in your PR. That is just used here to demonstrate the suggested change in markdown.